cmd/iriscli/main.go

```diff
@@ -152,6 +153,28 @@ func main() {
 		iserviceCmd,
 	)
 
+	//Add iservice1 commands
+	iservice1Cmd := &cobra.Command{
+		Use:   "iservice1",
```

Comment:
I think we should provide only one `iservice` cmd which has different behavior under the new version and old version iservice module.